samples/subsys/settings/boards/cc1352p7_lp.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_NVS=y
```

Comment:
Is this really a good single board default? I'd rather propose we make this the default for all TI SoCs that support flash/nvs (probably all of them CC @vaishnavachath), i.e. rename the file accordingly.

Reply:
Do you mean renaming and moving the file to socs ?
Something like socs/cc1352p7.conf (and the same for the other variant) ?

Reply:
Yes, there should now be a way to define [per-soc overlays](https://docs.zephyrproject.org/latest/build/kconfig/setting.html#the-initial-configuration) (in your case in `samples/subsys/settings/soc`):

> Furthermore, applications can have SoC overlay configuration that is applied to it, the file `socs/<SOC>_<BOARD_QUALIFIERS>.conf` will be applied if it exists, after the main project configuration has been applied and before any board overlay configuration files have been applied.

I've not yet tried this myself, but I guess it just works.